apps/students/views.py

```diff
@@ -216,8 +217,10 @@ def ajax_mark_homework(request):
     if request.method == 'POST' and request.is_ajax():
         stu_id = request.POST.get('stu_id')
         is_homework_done = request.POST.get('is_homework_done')
+        todayDate = request.POST.get('date')
+        todayDate = datetime.strptime(todayDate, "%a, %d %b %Y %H:%M:%S %Z")
```

Comment:
See `# Date and section selected before pressing submit button` section in `update_cwhw` function to know how to do it properly.

Reply:
Can you elaborate it, i didn't get it?

Reply:
Talking about this..

![image](https://user-images.githubusercontent.com/39924567/113025987-308bd080-91a6-11eb-8610-63d1967cfd36.png)